prod/westeurope/notifications/functions_pushnotif/function_app/terragrunt.hcl

```diff
@@ -103,6 +104,7 @@ inputs = {
     FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
     FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
+    TEST_LOGIN_FISCAL_CODES = test_users = join(",", local.testusersvars.test_users_internal_load)
```

Comment:
This should be `FISCAL_CODE_NOTIFICATION_BLACKLIST` instead of `TEST_LOGIN_FISCAL_CODES`

Reply:
Done